Method,Name,Error,Occurrences
GET,/supply/callcreatesupply,"HTTPConnectionPool(host='127.0.0.1', port=4000): Max retries exceeded with url: /supply/callcreatesupply (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f88334dc160>: Failed to establish a new connection: [Errno 111] Connection refused'))",2987
